### ISSUE TITLE
SMS messages are truncated on colons

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#342]
```
SMS messages that contain a colon (':') truncate the message at the position of 
this character.
```

Original issue reported on code.google.com by `alexande...@gmail.com` on 12 Nov 2013 at 1:34
- Merged into: #339

**Comment by GoogleCodeExporter** [comment]
```
Which version ?
It has been fixed in the last version 4.4.3. See 
https://code.google.com/p/gtalksms/issues/detail?id=339

```

Original comment by `Florent....@gmail.com` on 12 Nov 2013 at 1:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 13 Nov 2013 at 10:01
- Changed state: **Duplicate**
